hawkbit-repository/hawkbit-repository-jpa/src/main/java/org/eclipse/hawkbit/repository/jpa/utils/QuotaInputStream.java

```diff
@@ -0,0 +1,84 @@
+/**
+ * Copyright (c) 2019 Bosch Software Innovations GmbH and others.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.eclipse.hawkbit.repository.jpa.utils;
+
+import java.io.FilterInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+
+import org.eclipse.hawkbit.repository.exception.QuotaExceededException;
+
+public class QuotaInputStream extends FilterInputStream {
```

Comment:
Please avoid misunderstandings with the class name. Suggestion: FileSizeAndStorageQuotaCheckingInputStream would be a better name?